components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1/src/main/resources/publisher-api.yaml

```diff
@@ -8753,6 +9058,141 @@ definitions:
           $ref: '#/definitions/Tenant'
       pagination:
         $ref: '#/definitions/Pagination'
+  #----------------------------------------------
+  # The Alert list resource.
+  #----------------------------------------------
+  AlertTypesList:
+    title: Alert Types List
+    properties:
+      count:
+        type: integer
+        description: The number of alerts
+        example: 3
+      alerts:
+        type: array
+        items:
+          $ref: '#/definitions/AlertType'
+
+  #----------------------------------------------
+  # The AlertType resource.
+  #----------------------------------------------
+  AlertType:
+    title: Alert Type
+    properties:
+      id:
+        type: integer
+        description: The alert Id
+        example: 1
+      name:
+        type: string
+        description: The name of the alert.
+        example: "AbnormalRequestsPerMin"
+      isRequireConfiguration:
```

Comment:
Shall we rename this to requireConfiguration: true/false

Reply:
Fixed in https://github.com/wso2/carbon-apimgt/pull/7117

Reply:
Fixed in https://github.com/wso2/carbon-apimgt/pull/7117